docs/node/CATEGORIES/01-blockchain.md

```diff
@@ -289,6 +289,18 @@ interface MempoolInfo {
 }
 ```
 
+## getMempoolAncestors
+
+Get all in-mempool ancestors if txId is in mempool as string[] if verbose is false else as json object.
+
+```ts title="client.blockchain.getBlockHeader()"
+interface blockchain {
+  getMempoolAncestors (txId: string, verbosity: true): Promise<MempoolTx>
+  getMempoolAncestors (txId: string, verbosity: false): Promise<string[]>
+  getMempoolAncestors (txId: string, verbosity: true | false): Promise<MempoolTx | string[]>
```

Comment:
We should not use true | false.
We should use boolean.
Try match with the one you wrote in your RPC file